### ISSUE TITLE
Woka customization is slow

### ISSUE THREAD

**Issue by michaelWuensch** [thecodingmachine/workadventure#1805]
When a user customizes the woka it is really slow. It is not fun to cusomize a woka like this. 
The only way to go through the very long list of parts is by pressing the arrow keys. And then you have to wait every single time for response.

This sould feel a lot more fluent.

(Tested on Firefox, Windows)

**Comment by gparant** [comment]
Thanks for your feedback 💪

**Comment by tabascoeye** [comment]
I agree, at least a "key repeat" should be supported (i.e. when you hold the "right" arrow, it should continue to go to the next texture)
